src/ui/server/controllers/insurance/all-sections/index.ts

```diff
@@ -2,40 +2,31 @@ import { PAGES } from '../../../content-strings';
 import { ROUTES, TEMPLATES } from '../../../constants';
 import { Request, Response } from '../../../../types';
 import insuranceCorePageVariables from '../../../helpers/page-variables/core/insurance';
-import api from '../../../api';
 import generateGroupsAndTasks from '../../../helpers/task-list/generate-groups-and-tasks';
 import generateTaskList from '../../../helpers/task-list';
 import flattenApplicationData from '../../../helpers/flatten-application-data';
 
 export const TEMPLATE = TEMPLATES.INSURANCE.ALL_SECTIONS;
 
-export const get = async (req: Request, res: Response) => {
-  const { referenceNumber } = req.params;
+export const get = (req: Request, res: Response) => {
+  const { application } = res.locals;
```

Comment:
should it be res?

Reply:
yes 😄 the new middleware passes the application via `res.locals`